src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,220 @@
+
+### Example 2
+
+``` js
+// entry.js
+
+// dynamically import a.js and b.js
+import("./a");
+import("./b");
+```
+
+``` js
+// a.js
+import "./helpers"; // helpers is 40kb in size
+
+// ...
+```
+
+``` js
+// b.js
+import "./helpers";
+import "./more-helpers"; // more-helpers is also 40kb in size
+
+// ...
+```
+
+**Result:** A separate chunk would be created containing `./helpers` and all dependencies of it. At the import calls this chunk is loaded in parallel to the original chunks.
+
+Why:
+
+* Condition 1: The chunk is shared between both import calls
+* Condition 2: `helpers` is bigger than 30kb
+* Condition 3: Number of parallel requests at the import calls is 2
+* Condition 4: Doesn't affect request at initial page load
+
+Putting the content of `helpers` into each chunk will result into its code being downloaded twice. By using a separate chunk this will only happen once. We pay the cost of an additional request, which could be considered a tradeoff. That's why there is a minimum size of 30kb.
+
+T> With the new `optimizations.splitChunks.chunks: "all"` option the same would happend for initial chunks. Chunks can even be shared between entrypoints and on-demand loading.
+
+## Configuration
+
+For these people that like to have more control over this functionality, webpack provides a set of options to better fit your needs.
+
+W> If you are manually changing the split configuration, measure the impact of the changes to see and make sure there's a real benefit. The defaults are choosen to fit best practices of web performance.
+
+### Configuring cache groups
+
+The defaults assigns all modules from `node_modules` to a cache group called `vendors` and all modules duplicated in at least 2 chunks to a change group `default`.
+
+A module can be assigned to multiple cache groups. The optimization then prefers the cache group with the higher `priority` (`priority` option) or that one that forms bigger chunks.
+
+### Conditions
+
+Modules from the same chunks and cache group will form a new chunk when all conditions are fullfilled.
+
+There are 4 options to configure the conditions:
+
+* `minSize` (default: 30000) Minimum size for a chunk.
+* `minChunks` (default: 1) Minimum number of chunks that share a module before splitting
+* `maxInitialRequests` (default 3) Maximum number of parallel requests at an entrypoint
+* `maxAsyncRequests` (default 5) Maximum number of parallel requests at on-demand loading
+
+### Naming
+
+To control the chunk name of the split chunk the `name` option can be used.
+
+W> When assigning equal names to different split chunks they are merged together. This can be used to put all vendor modules into a single shared chunk, but it's not recommend since it can result in more code downloaded.
+
+The magic value `true` automatically chooses a name based on chunks and cache group key. Elsewise a string or function can be passed.
```

Comment:
s/Elsewise/Otherwise

[Elsewise usage in english](https://www.collinsdictionary.com/us/dictionary/english/elsewise)